src/asynchronous/client.rs

```diff
@@ -35,8 +35,8 @@ pub struct Client {
 }
 
 impl Client {
-    pub fn connect(path: &str) -> Result<Client> {
-        let fd = unsafe { client_connect(path)? };
+    pub fn connect(sockaddr: &str) -> Result<Client> {
+        let fd = unsafe { client_connect(sockaddr)? };
```

Comment:
I'd like to understand how we're going to handle this incompatible change since this change won't tell users a compile time that something changed. They'll have to wait until runtime, which is not ideal.

See my comments on #119.

Reply:
I think we can control it by feature(like `normal_unix_socket`) like the first version of this PR.
Default all only abstract socket is supported, if users enabled the new feature, it will support both abstract and normal sockets by the address pattern.

Reply:
Thanks @liubin - that sounds good to me.